### ISSUE TITLE
[BUG] error: externally-managed-environment 

### ISSUE THREAD

**Issue by Deekee62** [raspibolt/raspibolt#1312]
### Describe the bug
Since then I've upgraded to Debian Bookworm and when trying to use the OpenTimestamps client I get the following error:

```
ots --version
Traceback (most recent call last):
  File "/usr/local/bin/ots", line 5, in <module>
    from otsclient.ots import main
ModuleNotFoundError: No module named 'otsclient'
```

### Location in guide
[https://raspibolt.org/guide/bitcoin/bitcoin-client.html#opentimestamps-client]

### Expected behavior
I installed the open opentimestamps-client on Debain BullsEye some months ago using:

`$ sudo apt-get install python3 python3-dev python3-pip python3-setuptools python3-wheel`

and

`$ sudo pip3 install opentimestamps-client`

I also checked the OpenTimestamps client version to check that it is properly installed

```
$ ots --version
v0.7.1
```
An everything seemed to work fine and I used it many times to verify more timestamps, when installing additional programs (e.g. LND) and when updating existing programs to a newer version.

### Environment
 - Hardware platform: Apple Macmini6,2
 - Operating system: Debian 12
 - Version: No LSB modules are available. Debian GNU/Linux 12 (bookworm)

**Comment by Deekee62** [comment]
Are you sure --break-system-packages as mentioned in [#1305](https://github.com/raspibolt/raspibolt/issues/1305#issuecomment-1624404907) is the correct way do do it, as pip may break Python modules that part of the Debian system depends on.

I think it's recommend to install it with pipx (in the "pipx" Debian
package). as pipx will set up an environment isolated from other applications and
system Python modules, and install the application and its dependencies into
that.

**Comment by Deekee62** [comment]
It seems there is an open request to package it officially. But there hasn't been any
progress in a few years: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921251

**Comment by Deekee62** [comment]
The Debian provided python3 interpreter packages (python3.11 and pypy3) are now marked as being externally-managed, following [PEP-668](https://peps.python.org/pep-0668/). The version of python3-pip provided in Debian follows this, and will refuse to manually install packages on Debian's python interpreters, unless the --break-system-packages option is specified.

**Comment by Deekee62** [comment]
Finally I did a reinstall using:

 `$ sudo apt-get install python3 python3-dev python3-pip python3-setuptools python3-wheel `
 
 and then 
 
 `$ sudo pip3 install opentimestamps-client --break-system-packages`.

At the end of the installation I got the following warning:

**WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv**

But nevertheless OpenTimestamps client reported that it's properly installed again.

`$ ots --version
v0.7.1
`
 It seems to work fine and no problems so far.
